march_hardware/include/march_hardware/MarchRobot.h

```diff
@@ -50,6 +50,9 @@ class MarchRobot
 
   bool isEthercatOperational();
 
+  bool getTrainReturned();
```

Comment:
This method can be `const`
```suggestion
  bool getTrainReturned() const;
```
(Same as all other getters you added)

Reply:
The getters no long exist :+1: